packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -107,6 +107,9 @@ function showHelp() {
     appName                   the name of the application created by some presets  
 
     cli                       CLI to power the Nx workspace (options: "nx", "angular")
+    
+    style                     style option for determine default style schematics when no empty preset selected 
```

Comment:
`default style option to be used when a non-empty preset is selected`

Reply:
updated